apps/11gist-blog/script.js

```diff
@@ -1,5 +1,51 @@
 'use strict';
+marked.setOptions({
+    renderer: new marked.Renderer(),
+    gfm: true,
+    tables: true,
+    breaks: false,
+    pedantic: false,
+    sanitize: false,
+    smartLists: true,
+    smartypants: false
+});
 
 $(document).ready(function() {
-  // You code here
+    $.ajax('https://api.github.com/users/RocketShipp/gists', {
+        success: function(response) {
+            response.forEach(function(gist) {
+                var $myDescription = gist.description.slice(6, gist.description.length);
```

Comment:
Nitpicky stuff: here, `6` is what we call a [magic number](http://stackoverflow.com/questions/47882/what-is-a-magic-number-and-why-is-it-bad)

You could assign six to a variable that has an awesome, descriptive name